### PR TITLE
Firmware bios-256k.bin's reproducible materials

### PR DESCRIPTION
**Pull request by Xynnn007**
rootfs, or so-called OS image reproducible build materials. Including

* RBCI's Dockerfile
* bios reproducible builder
* readme

### CODE REVIEW

File: rbi/bios-256k/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+ENV MAKE_VERSION    1:4.2.1-10
+ENV GCC_VERSION     8.4.1-1
+
+RUN [ -n "$http_proxy" ] && sed -i '$ a proxy='$http_proxy /etc/dnf/dnf.conf ; true
+RUN dnf install -y make-$MAKE_VERSION \
+                gcc-$GCC_VERSION \
+                git \
+                python2
+RUN ln /usr/bin/python2 /usr/bin/python
```
Comment (hustliyilin) [inline rbi/bios-256k/Dockerfile:13]: It's better to use `alternatives --set python /usr/bin/python2`. You can refer to [the Dockerfile](https://github.com/occlum/occlum/blob/master/tools/docker/Dockerfile.centos8.2) in the Occlum.

图片

File: rbi/bios-256k/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# RB for bios-256k.bin
+
+## files
+`build-docker-image.sh` To build RBCI for bios
```
Comment (hustliyilin) [inline rbi/bios-256k/readme.md:4]: add an empty line please

Comment (hustliyilin) [inline rbi/bios-256k/readme.md:4]: add an empty line please

File: rbi/bios-256k/readme.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# RB for bios-256k.bin
+
+## files
```
Comment (hustliyilin) [inline rbi/bios-256k/readme.md:3]: Please add am empty line between the chapter title and content.

File: rbi/bios-256k/readme.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+`build-docker-image.sh` To build RBCI for bios
+`build-bios.sh` To build bios and check artifest
+
+## Steps
```
Comment (hustliyilin) [inline rbi/bios-256k/readme.md:7]: ditto

File: rbi/readme.md [hunk 5]
```diff
@@ -83,4 +83,17 @@ $cat report/kernel_report
 ===KERNEL RB REPORT===
 [Time] 2021-07-23 17:58:59
 [SUCCESSFUL] Same hash
+```
+
+### RB for bios-256k.bin
```
Comment (hustliyilin) [inline rbi/readme.md:91]: ditto

File: rbi/readme_cn.md [hunk 6]
```diff
@@ -26,4 +26,69 @@
 清除所有临时文件
 ```bash
 ./rbi clean
+```
+
+### kata-containers.img的可重复构建
```
Comment (hustliyilin) [inline rbi/readme_cn.md:31]: ditto

File: rbi/readme_cn.md [hunk 7]
```diff
@@ -26,4 +26,69 @@
...
+
+如果完整性校验通过，则可以看到报告如下
+```plaintext
+$cat report/kernel/kernel_report 
+===KERNEL RB REPORT===
+[Time] 2021-07-23 17:58:59
+[SUCCESSFUL] Same hash
+```
+
+### Kata VM使用BIOS的可重复构建
```
Comment (hustliyilin) [inline rbi/readme_cn.md:79]: ditto

File: rbi/bios-256k/readme.md (hunk 3 above)
Comment (Xynnn007) [reply rbi/bios-256k/readme.md:3]: thanks, have added

File: rbi/bios-256k/readme.md (hunk 4 above)
Comment (Xynnn007) [reply rbi/bios-256k/readme.md:8]: thanks, as mentioned, have added

File: rbi/readme.md (hunk 5 above)
Comment (Xynnn007) [reply rbi/readme.md:91]: thanks, as mentioned, have added

File: rbi/readme_cn.md (hunk 6 above)
Comment (Xynnn007) [reply rbi/readme_cn.md:34]: thanks, as mentioned, have added

File: rbi/readme_cn.md (hunk 7 above)
Comment (Xynnn007) [reply rbi/readme_cn.md:83]: thanks, as mentioned, have added

File: rbi/bios-256k/Dockerfile (hunk 1 above)
Comment (Xynnn007) [reply rbi/bios-256k/Dockerfile]: Thanks! It's a good way. fixed.

**Review by hustliyilin — APPROVED** [review]

**Pull request merged**